src/carplay.cs

```diff
@@ -940,5 +940,72 @@ interface CPAlertTemplate {
 		[Export ("actions", ArgumentSemantic.Strong)]
 		CPAlertAction [] Actions { get; }
 	}
+
+	[NoWatch, NoTV, NoMac, iOS (13,4)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface CPDashboardButton : NSSecureCoding {
+
+		[Export ("initWithTitleVariants:subtitleVariants:image:handler:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string[] titleVariants, string[] subtitleVariants, UIImage image, [NullAllowed] Action<CPDashboardButton> handler);
+
+		[Export ("image")]
+		UIImage Image { get; }
+
+		[Export ("titleVariants")]
+		string [] TitleVariants { get; }
+
+		[Export ("subtitleVariants")]
+		string [] SubtitleVariants { get; }
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (13,4)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface CPDashboardController {
+
+		[Export ("shortcutButtons", ArgumentSemantic.Copy)]
+		CPDashboardButton [] ShortcutButtons { get; set; }
+	}
+
+	interface ICPTemplateApplicationDashboardSceneDelegate { }
+
+	[NoWatch, NoTV, NoMac, iOS (13,4)]
+	[Protocol, Model (AutoGeneratedName = true)]
+	[BaseType (typeof (NSObject))]
+	interface CPTemplateApplicationDashboardSceneDelegate : UISceneDelegate {
+
+		[Export ("templateApplicationDashboardScene:didConnectDashboardController:toWindow:")]
+		void DidConnectDashboardController (CPTemplateApplicationDashboardScene templateApplicationDashboardScene, CPDashboardController dashboardController, UIWindow window);
+
+		[Export ("templateApplicationDashboardScene:didDisconnectDashboardController:fromWindow:")]
+		void DidDisconnectDashboardController (CPTemplateApplicationDashboardScene templateApplicationDashboardScene, CPDashboardController dashboardController, UIWindow window);
```

Comment:
Same as the previous comment.
```suggestion
		void DidDisconnect (CPTemplateApplicationDashboardScene templateApplicationDashboardScene, CPDashboardController dashboardController, UIWindow window);
```

Reply:
same as above `DidDisconnectDashboardController` of a `DashboardScene`